README.md

```diff
@@ -39,6 +39,67 @@ npm install
 npm run lint
 ```
 
+### Language mode
```

Comment:
This part of the README is getting quite long... can we pull it out into its own STYLE/GUIDELINE md file? And include other related headers. Make README more scannable and friendly